metadata.json

```diff
@@ -80,6 +80,6 @@
     }
   ],
   "dependencies": [
-  
+      {"name":"puppetlabs/stdlib","version_requirement":">= 4.6.0 <5.0.0"}
```

Comment:
Adding a dependency is considered a backwards-incompatible change normally I think, though basically everyone has stdlib installed...